### ISSUE TITLE
RedisSearch SORTBY does not work 

### ISSUE THREAD

**Issue by bertrand-riviere** [redis/node-redis#1749]
It might actually not be a "bug" but when I use the `SORTBY` search option it seems to be ignored (which makes sense considering it is commented out in /packages/search/lib/commands/SEARCH.ts).
I can see that a `MSORTBY` option is supported but when I try to use it I get an error from Redis (ReplyError: Unknown argument `MSORTBY` at position 1 for ...)
Is there a way to workaround this issue?

**Environment:**
 - **Node.js Version**: 14.17.0
 - **Redis Server Version**: 6.2.3
 - **Node Redis Version**: 4.0.0
 - **Platform**: Windows 10

**Comment by leibale** [comment]
You can use `client.sendCommand` with `SEARCH.transformArguments` and `SERACH.transformReply` as a workaround for now:
```javascript
import { transformArguments, transformReply } from '@node-redis/client/dist/lib/commands/SEARCH';

const searchArgs = transformArguments(...);
searchArgs.push('SORTBY', '...');
const reply = transformReply(await client.sendCommand(searchArgs));
```

the `SORTBY` option should be supported, It'll be released with `@node-redis/search@1.0.1`.

**Comment by bertrand-riviere** [comment]
Wahoo, that's exactly what I've done and I was about to comment my own issue in case it could be useful to someone else! ;)
(I still don't get the point of `MSORTBY` since it is not supported by Redis as far I know but maybe I've missed something..)

Many thanks anyway!

**Comment by amansubhan** [comment]
> You can use `client.sendCommand` with `SEARCH.transformArguments` and `SERACH.transformReply` as a workaround for now:
> 
> ```js
> import { transformArguments, transformReply } from '@node-redis/client/dist/lib/commands/SEARCH';
> 
> const searchArgs = transformArguments(...);
> searchArgs.push('SORTBY', '...');
> const reply = transformReply(await client.sendCommand(searchArgs));
> ```
> 
> the `SORTBY` option should be supported, It'll be released with `@node-redis/search@1.0.1`.

Can you explain this example for typescript? I am getting an error on the import statement 
`Cannot find module '@node-redis/client/dist/lib/commands/SEARCH' or its corresponding type declarations. ts(2307)`

**Comment by leibale** [comment]
@amansubhan I've updated my command

**Comment by bertrand-riviere** [comment]
> > You can use `client.sendCommand` with `SEARCH.transformArguments` and `SERACH.transformReply` as a workaround for now:
> > ```js
> > import { transformArguments, transformReply } from '@node-redis/client/dist/lib/commands/SEARCH';
> > 
> > const searchArgs = transformArguments(...);
> > searchArgs.push('SORTBY', '...');
> > const reply = transformReply(await client.sendCommand(searchArgs));
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > the `SORTBY` option should be supported, It'll be released with `@node-redis/search@1.0.1`.
> 
> Can you explain this example for typescript? I am getting an error on the import statement `Cannot find module '@node-redis/client/dist/lib/commands/SEARCH' or its corresponding type declarations. ts(2307)`

I'm not using TS but here is the import I use (and that works):
import redisSearch from '@node-redis/search';
then redisSearch.search.transformArguments for instance..

**Comment by amansubhan** [comment]
@bertrand-riviere @leibale Thanks to both of you, it's working now as you pointed out the import

> > > You can use `client.sendCommand` with `SEARCH.transformArguments` and `SERACH.transformReply` as a workaround for now:
> > > ```js
> > > import { transformArguments, transformReply } from '@node-redis/client/dist/lib/commands/SEARCH';
> > > 
> > > const searchArgs = transformArguments(...);
> > > searchArgs.push('SORTBY', '...');
> > > const reply = transformReply(await client.sendCommand(searchArgs));
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > the `SORTBY` option should be supported, It'll be released with `@node-redis/search@1.0.1`.
> > 
> > 
> > Can you explain this example for typescript? I am getting an error on the import statement `Cannot find module '@node-redis/client/dist/lib/commands/SEARCH' or its corresponding type declarations. ts(2307)`
> 
> I'm not using TS but here is the import I use (and that works): import redisSearch from '@node-redis/search'; then redisSearch.search.transformArguments for instance..
